elm-compiler.cabal

```diff
@@ -197,8 +199,8 @@ Test-Suite compiler-tests
         ansi-terminal >= 0.6.2.1 && < 0.7,
         base >=4.2 && <5,
         binary >= 0.7.0.0 && < 0.8,
-        blaze-html >= 0.5 && < 0.8,
-        blaze-markup >= 0.5.1 && < 0.7,
+        blaze-html >= 0.5 && < 0.9,
+        blaze-markup >= 0.5.1 && < 0.8,
```

Comment:
Is this just lining things up with other things in this file? If so, can you link to the line numbers?